package.json

```diff
@@ -44,6 +44,7 @@
     "errs": "~0.3.2",
     "feedsme-api-client": "^2.0.0",
     "fingerprinting": "~1.0.0",
+    "from2": "^2.3.0",
```

Comment:
FYI `minipass` is a new(ish) implementation that's quite a bit faster than through2: 

- [minipass repo](https://github.com/isaacs/minipass#readme)
- [minpass benchmarks](https://docs.google.com/spreadsheets/d/1oObKSrVwLX_7Ut4Z6g3fZW-AX1j1-k6w-cDsrkaSbHM/edit#gid=0)